floaters/src/main/java/com/mdg/droiders/floaters/ExpandedWindow.java

```diff
@@ -83,6 +83,7 @@ public boolean onTouch(View v, MotionEvent event) {
                                 container.setInitialPos(lp.leftMargin, lp.topMargin);
                                 //get the touch location
                                 container.setInitialTouchPos(event.getRawX(), event.getRawY());
+                                closeButtonLayout.setVisibility(View.VISIBLE);
```

Comment:
\[Checkstyle\] WARNING: 'block' child have incorrect indentation level 32, expected level should be one of the following: 24, 26, 28\.